medical_appointment/data/medical_appointment_sequence.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015-TODAY
+	     LasLabs [https://laslabs.com]
+	     Komit [komit-consulting.com]
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
-<!--
-    Copyright (C) 2015-TODAY LasLabs [https://laslabs.com]
-    @author Dave Lasley <dave@laslabs.com>
-    @license AGPL-3
--->
-
-<openerp>
+<odoo>
```

Comment:
OCA/maintainer-tools#253. I notice you pointed to the contribution docs and not the template module though. What was your thought on how to change that? Or did you just want to submit the PR yourself?